contracts/scripts/deploy.js

```diff
@@ -44,89 +33,73 @@ const save = (value, field, subfield = undefined) => {
   fs.writeFileSync(fileName, JSON.stringify(parameters));
 };
 
+// Send 0.01 ETH to oneself and wait for it to be mined, bumping the wallet's nonce
+const doBumpNonceTx = async (wallet) => {
+  const tx = await wallet.sendTransaction({
+    to: wallet.address,
+    value: ethers.utils.parseEther('0.01'),
```

Comment:
You could just send zero ETH, which I think is more common and clear. But doesn't make a difference